plugin/bitrise/execer.go

```diff
@@ -72,6 +73,17 @@ func (e *Execer) Exec(ctx context.Context) error {
 		}
 	}
 
+	// create the .envstore.yml file if not present
+	if !Is(e.Source, envStoreFile) {
```

Comment:
we should use `Is` to determine if a plugin is a Bitrise plugin.  If we are already executing the Bitrise plugin, we should not need to invoke `Is`.  Perhaps this should be a new / different helper function?